addons/web/static/src/xml/base.xml

```diff
@@ -1228,7 +1228,7 @@
     </div>
 </t>
 
-<input t-name="SearchView.InputView" type="text" class="o_searchview_input" accesskey="Q" placeholder="Search..." role="searchbox" aria-haspopup="true"/>
+<input t-name="SearchView.InputView" type="search" class="o_searchview_input" accesskey="Q" placeholder="Search..." role="searchbox" aria-haspopup="true"/>
```

Comment:
I'm ok with this change but only in master.

Reply:
Are you sure this change violates stability guidelines. It's not impossible, but IMHO it's very unlikely that `[type="text"]` is used to find this element instead of just `.o_searchview_input`.

Also this is actually a fix, not a feature.